circle.yml

```diff
@@ -14,10 +14,11 @@ steps: &steps
   - restore_cache: *restore_cache
   - run: "echo '//registry.npmjs.org/:_authToken=${NPM_TOKEN}' > ./.npmrc"
   - run: npm install
-  - run: npm test
+  - run:
+      command: npm test -- --ci --testResultsProcessor="jest-junit"
+      environment:
+        JEST_JUNIT_OUTPUT: "reports/junit/js-test-results.xml"
   - save_cache: *save_cache
-  - store_test_results:
```

Comment:
I think this is still necessary when using Circle CI 2.0. They don't check the output directories automatically anymore.